requirements.yml

```diff
@@ -6,3 +6,7 @@ collections:
   - name: community.docker
     version: ">=3.0.2"
   - name: containers.podman
+  - name: azure.azcollection
+    version: ">=1,<2"
+  - name: community.crypto
+    version: ">=2,<3"
```

Comment:
Here is an explicit definition.  How about using 1.8.0 here?

https://github.com/ansible-community/molecule-plugins/blob/5ebdfc60e457d5303e45fb237840e6592b98715f/src/molecule_plugins/gce/driver.py#L174-L176

Reply:
Done